src/panoramic/cli/local/writer.py

```diff
@@ -35,6 +42,8 @@ def write(self, actionable: Actionable, *, package: Optional[str] = None, file_n
             return self.write_model(actionable, package=package, file_name=file_name)
         elif isinstance(actionable, PanoVirtualDataSource):
             return self.write_data_source(actionable, package=package)
+        elif isinstance(actionable, PanoField):
+            return self.write_field(actionable, file_name=file_name)
```

Comment:
Are we missing the package argument here?

Reply:
Yeah, didn't include it at first, forgot to add it. Good catch.

Reply:
Fixed in 99dcaa9.